src/tests/ov_helpers/ov_snippets_models/src/subgraph_matmul.cpp

```diff
@@ -12,9 +12,76 @@
 namespace ov {
 namespace test {
 namespace snippets {
+namespace {
+constexpr float i8_fq_il = -35.0172004;
+constexpr float i8_fq_ih = 34.7436294;
+constexpr float u8_fq_il = 0;
+constexpr float u8_fq_ih = 0.820726;
+
+std::shared_ptr<ov::Node> build_mm_second_input(const ov::element::Type& precision,
```

Comment:
Minor, I don't insist:
```suggestion
std::shared_ptr<ov::Node> create_matmul_weights(const ov::element::Type& precision,
```

Reply:
This builder creates not only weights, but activations too. So I think create_matmul_weights is a bit confusing name

Reply:
I renamed it to `make_matmul_b_input` to be aligned with other builders